### PR TITLE
Show attachments in modal view

### PR DESCRIPTION
**Pull request by sseliverstov**
[//]: # (
. Thank you so much for sending us a pull request! 
.
. Make sure you have a clear name for your pull request. 
. The name should start with a capital letter and no dot is required in the end of the sentence.
. To link the request with isses use the following notation: (fixes #123, fixes #321\)
.
. An example of good pull request names:
. - Add Russian translation (fixes #123\)
. - Add an ability to disable default plugins
. - Support emoji in test descriptions
)

### Context
[//]: # (
Describe the problem or feature in addition to a link to the issues
)

#### Checklist
- [x] [Sign Allure CLA][cla]
- [ ] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

File: allure-generator/src/main/javascript/layouts/tree/TreeLayout.js [hunk 1]
```diff
@@ -28,5 +29,13 @@ export default class TreeLayout extends AppLayout {
     onRouteUpdate(testGroup, testResult, testResultTab) {
         this.routeState.set('treeNode', {testGroup, testResult});
         this.routeState.set('testResultTab', testResultTab);
+
+        const attachment = router.getUrlParams().attachment;
+        if (attachment) {
+            this.routeState.set('attachment', attachment);
+        } else {
+            this.routeState.unset('attachment', {silent: true});
```
Comment (just-boris) [inline allure-generator/src/main/javascript/layouts/tree/TreeLayout.js]: Silent is usually an anti-pattern. Can you not to use this option?

Comment (just-boris) [inline allure-generator/src/main/javascript/layouts/tree/TreeLayout.js]: You can add a check into `highlightSelectedAttachment` method instead.

**Comment by just-boris** [comment]
Visual review

image

image

I don't like the current inline display. My suggestion

image

**Comment by just-boris** [comment]
And my constant reminder about tab content: Steps should be on the first tab. I think this change should be done the first, before all other improvements with inline attachments.

**Comment by sseliverstov** [comment]
@just-boris 
I am not sure that it is finally variant of layout for steps attachments. Your suggestion is interesting to, but in this iteration i save curent view.

**Comment by just-boris** [comment]
* We have never use this blue color for borders. It just looks like 💩 . I tried to save it as much as I could.
* Modal still doesn't close if you will click on the place bellow the modal

**Comment by sseliverstov** [comment]
@just-boris how about this?

**Comment by sseliverstov** [comment]
@just-boris i am done

**Pull request merged**